### ISSUE TITLE
How to handle Validation failure during import

### ISSUE THREAD

**Issue by bill-anderson** [devinit/opendevdata#87]
At this stage ...

1. Validate all space, time and data series values and count the number of failures for each.
2. Store the row number in the file of the first error.
3. Report: " 00 space values, 00 time values failed vaildation. First space value occurs in row 00. First time value occurs in row 00. Please fix your file and repeat the import."
4. Clear out loaded data and take user back to the start of the import wizard.

**Comment by miclovich** [comment]
When free, please prepare some datasets with erroneous data/input for this issue, #86 #85 and #84 
cc: @bill-anderson

**Comment by bill-anderson** [comment]
I will hold off further testing until you've looked at #84 , right?

**Comment by miclovich** [comment]
Correct. Work with the cleanest data sets you have. Please prepare the others for use-cases where possible known errors might occur (I'll test them first before you do, less heart attacks)

**Comment by bill-anderson** [comment]
See email
